snapcraft/projects.py

```diff
@@ -152,6 +154,27 @@ class Architecture(ProjectModel):
     build_to: Optional[Union[str, UniqueStrList]]
 
 
+class AptDeb(ProjectModel):
```

Comment:
if this model is moved to repo, it might make it easier to part with into a new package in the future

Reply:
I'll change the boundaries between repo and the rest of snapcraft to reduce coupling and make it easier to spin off craft-repositories.

Reply:
Repo changed to be more self-contained with less API surface. The API is now consistent with craft-parts (data is received as a dictionary and validated using a validator function provided by the package). This will become clear when the package repositories functionality is integrated with the rest of snapcraft.